pkg/products/monitoring/dashboards/criticalSLOAlerts.go

```diff
@@ -225,6 +226,7 @@ const MonitoringGrafanaDBCriticalSLOAlertsJSON = `{
 			"targets": [{
 				"expr": "sum(ALERTS{severity='critical', alertstate='firing'}) or vector(0)",
 				"format": "time_series",
+				"instant": false,
```

Comment:
Added in these instant fields even for places where they're not being used, I think it adds a bit more readability to the already busy Json, but I can take them out if we don't want them.

Reply:
Is this default value of `instant` false? If so, I'm just thinking if there is a need to add this to each `target` block. I would be thinking along the lines of why are we explicitly adding this variable with it's default value but not other doing the same for other possible values?

Reply:
Is the default value of `instant` false? Just thinking if we need to set it is so, as we would be explicitly setting the default value for a single variable, but not for other possible variables.

Reply:
Yes, the default value for instant is false, I get your point, in this case it's probably best to remove them if we're not using those, thanks for the feedback :+1: